pypeit/wavetilts.py

```diff
@@ -299,11 +318,138 @@ def trace_tilts(self, arcimg, lines_spec, lines_spat, thismask, slit_cen):
                                            spat_order=self.par['spat_order'],
                                            maxdev_tracefit=self.par['maxdev_tracefit'],
                                            sigrej_trace=self.par['sigrej_trace'])
+                                           #, debug_pca=True, show_tracefits=True)
 
         # Return
         self.steps.append(inspect.stack()[0][3])
         return trace_dict
 
```

Comment:
Can you send me a screenshot of what the continuum subtracted arc image looks like for VLT X-shooter  for the K-band. I just want a visual confirmation that this is working.

Reply:
Yeah, here's the debug plot for the dev suite data.  It's not great at the slit edges, and you can see that the assumption of a wavelength-independent scaling across the slit doesn't work great.  But the question is, is it good enough to trace the line centroids?

![nir_continuum](https://user-images.githubusercontent.com/15785898/68430151-67ea7f00-0164-11ea-995d-623a5162ed4b.png)

Reply:
This definitely looks good enough to me, although I didn't really follow how you got this from the code. So one option is to use this as a starting point, get the tilts, and then once you have the tilts do another iteration, which could do a better job since you have the tilts. That said, given that we are taking out a smooth continuum I'm not sure why the tilts matter. I'm happy with this as long as the tilts QA looks good. Can you show X-shooter tilts QA with and with the rm_continuum set?

Reply:
Yeah, I'll post a couple of the orders when the dev suite finishes.  I think an iteration could be helpful in terms of identifying lines to trace because you can extract the full slit and build S/N in the line detection.  But I agree that it shouldn't matter much to the continuum determination.

Reply:
Here are a few tilt QA plots for NIR XShooter:
![Arc_tilts_2d_A_1_01_S0000](https://user-images.githubusercontent.com/15785898/68624342-b44e0a00-052a-11ea-8818-72e228d7ba85.png)
![Arc_tilts_2d_A_1_01_S0009](https://user-images.githubusercontent.com/15785898/68624376-c2038f80-052a-11ea-8141-e5119f7fa92b.png)
![Arc_tilts_2d_A_1_01_S0015](https://user-images.githubusercontent.com/15785898/68624402-da73aa00-052a-11ea-9a0f-716677ff5c6c.png)
![Arc_tilts_spat_A_1_01_S0000](https://user-images.githubusercontent.com/15785898/68624418-e8292f80-052a-11ea-8d5a-179c1776366b.png)
![Arc_tilts_spat_A_1_01_S0009](https://user-images.githubusercontent.com/15785898/68624496-0f7ffc80-052b-11ea-808a-6e74a6e8b0ee.png)
![Arc_tilts_spat_A_1_01_S0015](https://user-images.githubusercontent.com/15785898/68624531-23c3f980-052b-11ea-9a9d-24505cf0abb1.png)
![Arc_tilts_spec_A_1_01_S0000](https://user-images.githubusercontent.com/15785898/68624552-2f172500-052b-11ea-9e33-f6cf0ad3c8c1.png)
![Arc_tilts_spec_A_1_01_S0009](https://user-images.githubusercontent.com/15785898/68624581-40603180-052b-11ea-80f7-e1229e4cc501.png)
![Arc_tilts_spec_A_1_01_S0015](https://user-images.githubusercontent.com/15785898/68624592-481fd600-052b-11ea-8e56-d3584ec5610e.png)